pages/common/views/layouts/default.jsx

```diff
@@ -37,12 +37,14 @@ const Layout = ({
       static: { ...rest, content, urls }
     })
     : {};
-
+  if (scripts.length) {
+    scripts = ['/public/js/common/bundle.js'].concat(scripts);
+  }
```

Comment:
I like this more than I was expecting to.

We just need to be careful that there's no global initialisation that ends up in the bundle.